src/main/resources/plugin.yml

```diff
@@ -29,3 +29,5 @@ commands:
 permissions:
     rb.public:
         default: true
+    rb.op:
```

Comment:
This permission is different from the one you used in the command, also it'd be nice to be able to let players lookup othe biomes without having to give them a *.op permission